test/e2e/network/service.go

```diff
@@ -3282,42 +3280,60 @@ var _ = SIGDescribe("ESIPP [Slow]", func() {
 			framework.Failf("Service HealthCheck NodePort still present")
 		}
 
-		endpointNodeMap, err := jig.GetEndpointNodes()
+		config := e2enetwork.NewNetworkingTestConfig(f, false, false)
+
+		epNodes, err := jig.GetEndpointNodeNames()
 		framework.ExpectNoError(err)
+		endpointNodeMap := make(map[string][]string)
```

Comment:
Given that we only store InternalIp, cannot we make this `map[string]string`?
Adding a line of comment what these to maps hold would be nice (map from node name to internal ip)